protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1687,9 +1687,12 @@ message ListTransfersRequest {
   // Restrict transfer to those where the given public key is a sender or receiver.
   optional string pubkey = 1;
   // Restrict transfers to those in the given direction from the supplied public key.
+  // When is_reward is true, and pubkey is set, then direction MUST be TRANSFER_DIRECTION_DIRECTION_FROM.
   TransferDirection direction = 2;
   // Optional pagination control
   optional Pagination pagination = 3;
+  // Optional filter to return only reward transfers.
```

Comment:
```suggestion
  // Restrict transfers to those related to reward payments.
```

Reply:
Is "payments" right here if this also includes transfers that add assets to a reward pool?

Reply:
I'll keep it as transfers. I'd argue a payment is when funds move between accounts, this is just a standing order to conditionally/periodically make such payments.

Reply:
It could be "Restrict results to only reward transfers", or "Filter to return only reward transfers"

Reply:
I have a slight preference for the first one. But consistency is best so whichever style we use more frequently should be the winner.

Reply:
Done